### ISSUE TITLE
Clarify differences with other frameworks?

### ISSUE THREAD

**Issue by corlaez** [przemyslawjanpietrzak/rembrandt#22]
It would be nice to know why should we use rembrandt instead of react, for example.

Would you consider adding a section like that?

**Comment by przemyslawjanpietrzak** [comment]
Good point! I prepared some explanation in https://github.com/przemyslawjanpietrzak/rembrandt/pull/23. Please tell me what you think

**Comment by corlaez** [comment]
I read the linked pr.
So Rembrant is a reasonml library.
It solves both state management and ui rendering right?

**Comment by przemyslawjanpietrzak** [comment]
That's correct
